### DISCUSSION TITLE
11.12-17.12.23

### DISCUSSION THREAD

**Discussion by carolina-cza** [carolina-cza/Tecna#4] [Weekly Blog]
Moin.
Die Woche war jetzt nicht die aktivste, was auch teils krankheits  und Vorlesungsplan bedingt war...
Am Anfang der Woche hab ich mir ein Tutorial durchgelesen wie man [tictactoe](https://github.com/carolina-cza/Tecna/blob/codespace-super-space-yodel-x55ppqjx97jqfp699/daily_stuff/TicTacToe.py) programmiert (das Tutorial war echt nicht schlecht).Auf dem ersten blick hat es auch super funktioniert, aber beim genaueren hinsehen hat der Code nicht wirklich das gemacht was er sollte. Ich vermute ich hab da ein Form/Syntax Fehler drinnen, wahrscheinlich werde ich es die Woche dann noch debuggen (hopefully). 
Aber naja passiert. 
Im weiteren hab ich mir die Woche auch noch Exception angeschaut und es ist mir immer noch ein halber Mythos wie ich die zum laufen bringen soll, aber wir arbeiten uns Stück für Stück weiter und hoffentlich hab ich es bis Weihnachten gemeistert ^^

**Comment by Massiv153** [comment]
👍

**Comment by katharinaheller** [comment]
Das Tutorial zum Programmieren von Tic Tac Toe klingt interessant, und es ist immer frustrierend, wenn der Code auf den ersten Blick gut aussieht, aber sich dann herausstellt, dass etwas nicht wie geplant funktioniert. Form- und Syntaxfehler sind manchmal echt hartnäckig, aber das Debuggen wird sicherlich helfen. Viel Erfolg dabei!
